packages/decision-tree/src/components/Question/questionTypes/Date/Date.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react';
+import { Box, DateInput, Flex } from '@cmpsr/components';
+import { DateProps, uiDateFormat, isoDateFormat } from './types';
+import { QuestionTitle } from '@components/Question/components/QuestionTitle';
+import { inputMargin } from '@components/Question/Question';
+import { parse, format, isValid } from 'date-fns';
+
+export const DateComp: FC<DateProps> = ({ data, saveAnswer, defaultValue }) => {
+  const { label, id, placeholder, whyWeAskExplanation } = data;
+
+  const handleChange = ({ currentTarget }) => {
+    const date = parse(currentTarget.value, uiDateFormat, new Date());
+    if (!isValid(date)) return;
+
+    const formattedDate: string = format(date, isoDateFormat);
+    saveAnswer({ type: 'date', value: formattedDate });
+  };
+
+  const defaultDate = defaultValue?.value ? format(defaultValue.value, uiDateFormat) : null;
+
+  return (
+    <Box>
+      <QuestionTitle question={label} whyWeAskExplanation={whyWeAskExplanation} />
+      <Flex px={inputMargin}>
+        <DateInput
+          key={id}
+          variant="outline"
+          placeholder={placeholder}
+          defaultValue={defaultDate}
+          onInput={handleChange}
+          mode="MM/DD/YYYY"
+          textTransform="uppercase"
```

Comment:
is this needed?

Reply:
Yes, design is in uppercase, and the inputdate component does not allow me to modify the text while typing, so it was very weird having the palceholder uppercase, and then while typing in lowercase. If you can thing of a different solution, please let me know